expo/gbe/models.py

```diff
@@ -344,6 +344,10 @@ def schedule(self):
             resources_allocated__resource__worker___item=self)]
         return sorted(set(events), key=lambda event: event.start_time)
 
+    def volunteer_schedule(self, conference=None):
+        conference = conference or Conference.current_conf()
```

Comment:
After seeing how many times in a thread we look up the current conference in a single request/response with the debug toolbar, I'm worried about the pattern.  We reuse it all the time, but I'd like us to consider picking where we choose the conference and then streamlining all the code so that the conference selection only happens once in a thread.  Not a blocker.